robotbase/src/main/java/frc/robot/Constants.java

```diff
@@ -98,6 +97,22 @@ public static final class ELEVATOR {
     public static final Distance[] ELEVATOR_HEIGHT_SETPOINTS = {};
   }
 
+  public static final class ALGAE_RUNNER {
```

Comment:
We'll want to add constants for the following:

- A boolean for if the motor is inverted
- The IdleMode of the motor
- Free and stall limit amps
- Ramp rate

We'll also need to update MOTOR_CONFIG with these new constants